### PR TITLE
Update launch.md for args to include comment about doublequoting

### PR DESCRIPTION
**Pull request by pieandcakes**
https://github.com/Microsoft/vscode-cpptools/issues/1787

@bandzaw Please let me know if this is clearer?

### CODE REVIEW

**Comment by pieandcakes** [comment]
@bobbrow for review

**Review by WardenGnaw — APPROVED** [review]

File: launch.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ Set or change the following options to control VS Code's behavior during debuggi
 The following options enable you to modify the state of the target application when it is launched:
 
 * #### `args`
-  JSON array of command line arguments to pass to the program when it is launched. Example `["arg1", "arg2]`.
+  JSON array of command line arguments to pass to the program when it is launched. Example `["arg1", "arg2"]`. If you are escaping characters you will need to double escape them. For example `["{///"arg1///": true}]` will send `{"arg1": true}` to your application.
```
Comment (sean-mcmanus) [inline launch.md]: Are these supposed to be backslashes?

Comment (pieandcakes) [reply launch.md]: yes that is correct. I need to fix it.

Comment (vbaurea72) [reply launch.md]: This isn't working. Double escaping is stil broken

Comment (pieandcakes) [reply launch.md]: @vbaurea72 can you open an issue with more detail?

**Pull request merged**